doc/Interns/t-phnico/MyFirstCommit.md

```diff
@@ -0,0 +1 @@
+May 5th, 2020
```

Comment:
Forgot to mention, want date AND time in this file.  Also, your merge should be to your team branch.

Reply:
Fixes made. Changed the base branch to my team and added the date (on line 2 of the file).